src/main/java/com/internet/shop/controllers/CompleteOrderController.java

```diff
@@ -0,0 +1,30 @@
+package com.internet.shop.controllers;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Order;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.servise.interfaces.OrderService;
+import com.internet.shop.servise.interfaces.ShoppingCartService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class CompleteOrderController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private final OrderService orderService =
+            (OrderService) injector.getInstance(OrderService.class);
+    private final ShoppingCartService shoppingCartService =
+            (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long shoppingCartUserID = Long.parseLong(req.getParameter("userId"));
```

Comment:
We need to use constant 1L for now

Reply:
agree

Reply:
later we will change it to userId from session